ludwig/models/llm.py

```diff
@@ -658,6 +672,16 @@ def save(self, save_path):
         else:
             logger.info("Skipped saving LLM without weight adjustments.")
 
+    def save_base_model(self, save_path):
+        """Saves the base LLM model to the given path."""
+        # TODO: see the "TODO" statement from "LLM.save()" in this module.
+        if self.config_obj.trainer.type != "none":
+            weights_save_path = os.path.join(save_path, MODEL_WEIGHTS_FILE_NAME)
+            self.model.base_model.save_pretrained(weights_save_path)
+            self.tokenizer.save_pretrained(weights_save_path)
```

Comment:
Would be nice if you could add a comment here on why the tokenizer is being saved (as we discussed)!

Reply:
@arnavgarg1 I will certainly do that -- in the next commit.  Thanks.